nuvla/job_engine/job/base.py

```diff
@@ -14,8 +14,9 @@
 
 STATSD_PORT = 8125
 
-names = ['Cartman', 'Kenny', 'Stan', 'Kyle', 'Butters', 'Token', 'Timmy', 'Wendy', 'M. Garrison',
-         'Chef', 'Randy', 'Ike', 'Mr. Mackey', 'Mr. Slave', 'Tweek', 'Craig']
+names = ['Cartman', 'Kenny', 'Stan', 'Kyle', 'Butters', 'Token', 'Timmy',
+         'Wendy', 'M. Garrison', 'Chef', 'Randy', 'Ike', 'Mr. Mackey',
+         'Mr. Worker', 'Tweek', 'Craig']
```

Comment:
It's not the right name. 
There is no "Mr. Worker".
cf: https://en.wikipedia.org/wiki/List_of_South_Park_characters#Recurring_characters

Reply:
I invented that character and I hope it's all right ;-)